pkg/manager/impl/node_execution_manager.go

```diff
@@ -437,9 +437,12 @@ func (m *NodeExecutionManager) GetNodeExecutionData(
 		logger.Debugf(ctx, "failed to transform node execution model [%+v] when fetching data: %v", request.Id, err)
 		return nil, err
 	}
-	signedInputsURLBlob, err := m.urlData.Get(ctx, nodeExecution.InputUri)
-	if err != nil {
-		return nil, err
+	signedInputsURLBlob := admin.UrlBlob{}
+	if nodeExecution.InputUri != "" {
```

Comment:
should we also update https://github.com/flyteorg/flyteadmin/blob/0387a06511b633b485dbc2d8f216569bd2c22239/pkg/manager/impl/util/data.go#L9 to check if the blob is not empty?

Reply:
Done